packages/client/src/generation/TSClient/Enum.ts

```diff
@@ -23,7 +23,10 @@ export class Enum implements Generatable {
 ${indent(type.values.map((v) => `${v}: ${this.getValueJS(v)}`).join(',\n'), TAB_SIZE)}
 }`
     const enumBody = this.isStrictEnum() ? `makeStrictEnum(${enumVariants})` : enumVariants
-    return `exports.${this.useNamespace ? 'Prisma.' : ''}${type.name} = ${enumBody};`
+
+    return this.useNamespace
+      ? `exports.Prisma.${type.name} = ${enumBody};`
+      : `exports.${type.name} = exports.$Enums.${type.name} = ${enumBody};`
```

Comment:
This seems like it brings a new feature to the runtime, should we document it, or is it only internal?

Reply:
Not sure how useful this feature would be to the general public, but we can indeed document it.